ci/Jenkinsfiles/release.groovy

```diff
@@ -95,24 +97,19 @@ pipeline {
       steps {
         container('maven') {
           script {
-            def jiraVersionName = "coldstorage-${VERSION}"
-            // create a new released version in Jira
-            def jiraVersion = [
-                project: 'NXP',
-                name: jiraVersionName,
-                description: "Cold Storage Addon ${VERSION}",
-                releaseDate: LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE),
-                released: true,
+            def jiraIssueFetchers = [
+                    type                 : 'jira',
+                    jql                  : "project = ${JIRA_PROJECT} and fixVersion = ${JIRA_MOVING_VERSION}",
+                    newJiraVersion       : [
+                            project    : env.JIRA_PROJECT,
+                            name       : env.JIRA_RELEASED_VERSION,
+                            description: "Cold Storage Addon ${VERSION}",
+                            releaseDate: LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE),
+                            released   : true,
+                    ],
+                    jiraMovingVersionName: env.JIRA_MOVING_VERSION,
             ]
```

Comment:
```suggestion
            def jiraIssueFetchers = [
                type                 : 'jira',
                jql                  : "project = ${JIRA_PROJECT} and fixVersion = ${JIRA_MOVING_VERSION}",
                newJiraVersion       : [
                    project    : env.JIRA_PROJECT,
                    name       : env.JIRA_RELEASED_VERSION,
                    description: "Cold Storage Addon ${VERSION}",
                    releaseDate: LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE),
                    released   : true,
                ],
                jiraMovingVersionName: env.JIRA_MOVING_VERSION,
            ]
```

Reply:
It may be related to IDE configuration.

On a Groovy file, click on _Configure Indents for Groovy_:
<img width="413" alt="image" src="https://github.com/user-attachments/assets/372b30ea-8c8b-41b3-8f6f-8a90f23b5ce7" />

Then:
<img width="940" alt="image" src="https://github.com/user-attachments/assets/bb446903-8fc4-4d8b-90bd-4b1b1a2693fb" />